changelog/undistributed/changelog_show_access_list_20210614202815.rst

```diff
@@ -0,0 +1,8 @@
+--------------------------------------------------------------------------------
+                                New
+--------------------------------------------------------------------------------
+* ASA
+    * Added ShowAccessList:
+      * show access-list
```

Comment:
Please make sure the indentation is 4 spaces per indent. During the release process we use an automated script to generate the final changelog